### PR TITLE
`azurerm_cosmosdb_postgresql_cluster` - Mark `coordinator_storage_quota_in_mb` and `coordinator_vcore_count` as `optional`

### PR DESCRIPTION
**Pull request by neil-yechenwei**
fixes https://github.com/hashicorp/terraform-provider-azurerm/issues/23923

image

### CODE REVIEW

File: internal/services/cosmos/cosmosdb_postgresql_cluster_resource_test.go [hunk 1]
```diff
@@ -263,10 +263,11 @@ resource "azurerm_cosmosdb_postgresql_cluster" "test2" {
   source_location      = azurerm_cosmosdb_postgresql_cluster.test.location
   source_resource_id   = azurerm_cosmosdb_postgresql_cluster.test.id
   point_in_time_in_utc = azurerm_cosmosdb_postgresql_cluster.test.earliest_restore_time
+  node_count           = 0
```
Comment (stephybun) [inline internal/services/cosmos/cosmosdb_postgresql_cluster_resource_test.go:266]: why have we removed node count here?

Comment (neil-yechenwei) [reply internal/services/cosmos/cosmosdb_postgresql_cluster_resource_test.go:266]: I didn't removed "node_count". It's just the format adjusting.

**Comment by neil-yechenwei** [comment]
@stephybun , I replied your comment. Please take another look. Thanks.

**Review by stephybun — APPROVED** [review]
Thanks @neil-yechenwei LGTM 👍

**Pull request merged**